lua/entities/sent_spawnpoint/init.lua

```diff
@@ -128,37 +125,38 @@ function ENT:Initialize()
     self.Entity.LinkedPlayers = {}
 
     local phys = self.Entity:GetPhysicsObject()
-    if ( phys:IsValid() ) then
-        phys:Wake()
-        phys:EnableDrag(true)
-        phys:EnableMotion(false)
-    end
+    if not phys:IsValid() then return end
+    
+    phys:Wake()
+    phys:EnableDrag( true )
+    phys:EnableMotion( false )
 end
 
 function ENT:OnRemove()
     local effectdata1 = EffectData()
     effectdata1:SetOrigin( self.Entity:GetPos() )
     util.Effect( "spawnpoint_start", effectdata1, true, true )
 
-    unlinkAllPlayersFromSpawnPoint(self.Entity, {})
+    unlinkAllPlayersFromSpawnPoint( self.Entity, {} )
 end
 
 function ENT:Use( player, caller )
-    if ( player.LinkedSpawnPoint and player.LinkedSpawnPoint == self.Entity ) then
+    if player.LinkedSpawnPoint == self.Entity then
```

Comment:
I know this one seems pedantic but could you assign this to a variable too?

```lua
playerLinkedToSpawnPoint
```
or something. it would make reading a little easier.